packages/meditrak-server/src/routes/utilities/translateObjectFields.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+export const translateObjectFields = async (object, objectTranslators) => {
```

Comment:
Nice little util!

Reply:
I actually thought about putting this into `@tupaia/utils` where `ObjectValidator` sits. But for now it still feels a little too custom to me?

Reply:
Yeah...I think I agree with that. But keep it in mind when working across other packages of course!